.spellcheck.yaml

```diff
@@ -12,7 +12,10 @@ matrix:
   - pyspelling.filters.context:
       context_visible_first: true
       delimiters:
-      # ReST: code blocks
+      # RST directives
+      - open: '^\.{2}[ ]+[a-z-]+[:]{2}'
```

Comment:
@neon60 how do you think this will interact with the other code block sections that were added?

Reply:
I am testing locally. The code block check also not working correctly, I need more time to be able to answer the question.